models/slotModel.js

```diff
@@ -0,0 +1,42 @@
+const mongoose = require('mongoose');
```

Comment:
why did you repeat this file?
@HamdyMohamed10 has already written it.

and hence, the validator file should also be edited

Reply:
@HamdyMohamed10  slot model have been updated  so that i forgot to delete after add my new one 
i will add fix it now